packages/core/src/components/external/PlanningView/PlanningView.tsx

```diff
@@ -299,13 +333,21 @@ const getStyles = Colors =>
     circleButton: {
       marginHorizontal: 5,
       marginTop: 5,
-      width: 50 + 50 * (1 / 3),
+      width: 35 + 50 * (1 / 3),
     },
     headerPlanning: {
       flexDirection: 'row',
+      justifyContent: assignedToMeSwitch ? 'space-between' : 'flex-end',
       alignItems: 'center',
-      alignContent: 'flex-end',
-      alignSelf: 'flex-end',
+    },
+    headerButton: {
+      flexDirection: 'row',
+      alignItems: 'center',
+      marginRight: 10,
+    },
```

Comment:
There is more space on the right of the screen because in addition to `marginRight: 10` there is `marginHorizontal: 5` on `circleButton` style